resources/js/Pages/Cities/Index.vue

```diff
@@ -148,15 +150,15 @@ function toggleSortDialog() {
 
           <div class="mb-3 mt-4 flex flex-wrap items-center justify-end md:justify-between">
             <button class="mr-1 rounded bg-blumilk-500 px-5 py-3 text-sm font-medium text-white shadow-md hover:bg-blumilk-400 md:py-2" @click="toggleStoreDialog">
-              Create city
+              {{ $t('CRUD.Create_city') }}
```

Comment:
I think it will be better to have translations keys in `camelCase`

For example
`crud.createCity`

Reply:
Changed to camelCase